src/import/sst_service.rs

```diff
@@ -554,6 +554,15 @@ where
         self.threads.spawn_ok(buf_driver);
         self.threads.spawn_ok(handle_task);
     }
+
+    fn duplicate_detect(
```

Comment:
I don't think it's necessary as new grpcio will provide a default implementation.

Reply:
I have removed it.